src/tools/roc_recv/main.cpp

```diff
@@ -116,6 +122,13 @@ int main(int argc, char** argv) {
         }
     }
 
+    if (!args.max_frame_size_given) {
+        // 2 bytes per sample?
```

Comment:
I'm not sure if using 2 bytes per sample is correct... I'm assuming 16-bit PCM here, as I've seen that elsewhere.

Reply:
16-bit PCM is used for network packets (by default), but internal frames, for which we use frame pools and this limit, are 32-bit float.

Here I think we should use `sizeof(audio::sample_t)`.